### ISSUE TITLE
NVDA doesn't read the catagory 'Gradient fill' when user navigates to first control under 'Data bars' menu item. 

### ISSUE THREAD

**Issue by v-raipra** [nvaccess/nvda#7967]
User Experience:

Screen reader users will not get to know the heading of a particular list item if NVDA doesn't read it.

Test Environment:
Build: 8913.1000
OS: 17056.rs_prerelease.171205-1222
Screen reader: NVDA (2017.4)
[New Compressed (zipped) Folder.zip](https://github.com/nvaccess/nvda/files/1702937/New.Compressed.zipped.Folder.zip)

Repro Steps:

Switch On NVDA.
Open Excel.
Create a blank workbook.
Write something in the cells and select those cells.
Navigate to ‘Conditional formatting’ under ‘Home’ tab item using tab key and arrow keys and select it using enter key.
Navigate to ‘Data bars’ sub menu item using up/down arrow key and select it.
Navigate to the first list item and observe the NVDA speech.

Actual:
NVDA doesn't read the category 'Gradient fill' when user navigates to first control under 'Data bars' menu item. 

Expected:
NVDA should announce the category 'Gradient fill'/'Solid fill' so that the user gets to know under which heading the list item is present.

Note:
Same issue repro with Jaws.

**Comment by ehollig** [comment]
@v-raipra, did you mean to close this?
